rootex/common/common.h

```diff
@@ -47,3 +49,16 @@
 /// Logs file, function, message in red color
 #define ERR_CUSTOM(m_file, m_func, m_Msg) ::OS::PrintError(String(m_file) + ":" + String(m_func) + ": " + m_Msg);
 #endif // _DEBUG
+
+std::vector<std::string> &split(const std::string& s, char delim)
```

Comment:
The return value is ill-formed. Returning local addresses is actually undefined behavior. Also please use `std` types as imported in `types.h` i.e. `std::string` -> `String`.